components/dash-core-components/src/components/Checklist.react.js

```diff
@@ -198,4 +217,5 @@ Checklist.defaultProps = {
     value: [],
     persisted_props: ['value'],
     persistence_type: 'local',
+    inline: false,
```

Comment:
🔪

Reply:
@chriddyp what do think of having a standard `inline=True` shortcut like dash bootstrap components, that substitutes and is the equivalent of the more verbose `style = {'display': 'inline'}`